### PR TITLE
Srcset attribute

### PR DESCRIPTION
**Pull request by gasbriones**
Hi @xiaolin , here me again! I was working into a new and interesting feature for the image-gallery. Do you hear about srcSet? https://ericportis.com/posts/2014/srcset-sizes/.

The change that I did is about optimize image-gallery in different types of devices. For example, if you're using this gallery into a mobile with a high resolution you probably want to have a higher size image than a device with a lower resolution.
This is possible through the srcset and size attributes into img tag a new way to work with adaptive images.

Thanks a lot,
Greetings from Buenos Aires, Argentina.
Gaston.

### CODE REVIEW

File: example/app.js [hunk 1]
```diff
@@ -75,57 +75,33 @@ class App extends React.Component {
...
-        original: 'https://raw.githubusercontent.com/xiaolin/linxtion.github.io/master/static/img/image-gallery/11.jpg',
-        thumbnail: 'https://raw.githubusercontent.com/xiaolin/linxtion.github.io/master/static/img/image-gallery/11t.jpg'
-      },
-      {
-        original: 'https://raw.githubusercontent.com/xiaolin/linxtion.github.io/master/static/img/image-gallery/12.jpg',
-        thumbnail: 'https://raw.githubusercontent.com/xiaolin/linxtion.github.io/master/static/img/image-gallery/12t.jpg'
+        original: 'http://lorempixel.com/1600/900/nature/4/',
+        thumbnail: 'http://lorempixel.com/150/84/nature/4/',
+        srcSet:'http://lorempixel.com/1280/720/nature/4/ 1280w, http://lorempixel.com/1024/576/nature/4/ 1024w, http://lorempixel.com/800/450/nature/4/ 800w, http://lorempixel.com/360/203/nature/4/ 360w',
+        sizes:'100vw'
```
Comment (xiaolin) [inline example/app.js]: Id like to keep the images the way they were before. We can let the user read the documentation and see that `srcSet`, and `sizes` are supported

File: src/ImageGallery.jsx [hunk 2]
```diff
@@ -463,6 +463,8 @@ export default class ImageGallery extends React.Component {
             <img
               src={item.original}
               alt={item.originalAlt}
+              srcSet={item.srcSet}
+              sizes={item.sizes}
```
Comment (xiaolin) [inline src/ImageGallery.jsx]: This is a very nice feature!

**Comment by franciscok** [comment]
Hi guys! Intereseting feature.  I was wondering if you are thinking on deploy this changes into a new release. I'd like to add the srcSet and size to my project.
Thanks @xiaolin and @gasbriones.

**Comment by xiaolin** [comment]
@gasbriones  I can merge, and release this once `example/app.js` is reverted, Thanks.

**Comment by gasbriones** [comment]
Hi @xiaolin!! you need to send a new PR with revert in app.js file??

Regards!

**Pull request merged**